modules/stanford_layout_paragraphs/src/EventSubscriber/StanfordLayoutParagraphsSubscriber.php

```diff
@@ -19,6 +27,16 @@ public static function getSubscribedEvents(): array {
     ];
   }
 
+  /**
+   * Event subscriber constructor.
+   *
+   * @param \Drupal\Core\Layout\LayoutPluginManager $layoutPluginManager
+   *   Drupal core layout plugin manager.
+   */
+  public function __construct(LayoutPluginManager $layoutPluginManager) {
```

Comment:
Be sure to always use interfaces when they are available. `LayoutPluginManagerInterface` in this case.